en/docs/learn/customizing-automated-emails.md

```diff
@@ -179,6 +179,19 @@ Once you have set the template language for the user, it is also necessary to sp
     
         For example, use **fr\_fr** for the French locale, **pt\_br** for Brazil locale and **it\_it** for the Italian locale. This is the format used for country locales in the Identity Server.
         For more information on country locales, see [Locale codes](http://www.science.co.il/language/Locale-codes.php).
+
+    !!! note
+
+        If the local value is not defined for the user, the default locale value will be “en_US”. From the update level 250 you can change the default locale value using the following configuration,
+
+        ``` 
+        [identity_mgt.notification]
+        default_locale = "<locale>"
+        ```
+
+        If the above configuration is defined in the deployment.toml file, it will be used as the locale for both SMS and email flows; otherwise, the default value (en_US) will be assigned.
```

Comment:
```suggestion
        If the above configuration is added to the `deployment.toml` file, the specified value will be used as the locale for both SMS and email flows; otherwise, the default value (`en_US`) will be assigned.
```